slash/react/src/Divider/Divider.tsx

```diff
@@ -4,17 +4,18 @@ import "@axa-fr/design-system-slash-css/dist/Divider/Divider.css";
 
 type DividerProps = {
   mode?: "horizontal" | "vertical";
-  classModifier?: string;
+  className?: string;
 };
```

Comment:
pour quoi ne pas faire 
```suggestion
type DividerProps = ComponentProps<'hr'> & {
mode?: "horizontal" | "vertical";
};
```

et du coup en cas de besoin on peut même  {...props } sur le hr.

qu'en dis tu ?

Reply:
yes ça semble encore mieux

Reply:
done